eng/DependencyGraph/gen-dep-graph-data.ps1

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script requires Powershell 6 which defaults LocalMachine to Restricted on Windows client machines.
+# From a Powershell 6 prompt run 'Get-ExecutionPolicy -List' and if the LocalMachine is Restricted or Undefined then
+# run the following command from an admin Powershell 6 prompt 'Set-ExecutionPolicy -ExecutionPolicy RemoteSigned'. This
+# will enable running scripts locally in Powershell 6.
+
+# Use case: Generate the dependency graph data file (data.js) for track 2 (client) libraries.
```

Comment:
I assume you wrote this independent from Jonathan's dependency report tools because his didn't have the correct hooks to get the graph file we needed. Do you know if the report and this dependency graph are going have a consistent representation?

Reply:
I wrote this independent of Jonathan's dependency report because this is only getting first level dependencies which are directly in each of the pom files. Even if I were to use the output from the dependency report it would require some massive transformations/processing to get it into the format that the html page needs. Doing it this way was a bit more straightforward and easy to understand.